feat(kmp): update getting started and add manual setup page (#11497)
getsentry/sentry-docs: docs/platforms/kotlin-multiplatform/index.mdx

To view and resolve the recorded error, log into [sentry.io](https://sentry.io) and select your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.

## Next Steps

> - <PlatformLink to="/features">Learn more about Sentry's Kotlin Multiplatform SDK features</PlatformLink>
> - <PlatformLink to="/initialization-strategies">Learn about different strategies on how to initialize the SDK</PlatformLink>
> - <PlatformLink to="/configuration/gradle">Configuration options for the Kotlin Multiplatform Gradle Plugin</PlatformLink>
> - <PlatformLink to="/debug-symbols">Add readable stack traces to your errors</PlatformLink>

Reviewer: ```suggestion
- <PlatformLink to="/features">Learn more about Sentry's Kotlin Multiplatform SDK features</PlatformLink>
- <PlatformLink to="/initialization-strategies">Learn about different ways to initialize the SDK</PlatformLink>
- <PlatformLink to="/configuration/gradle">See configuration options for the Kotlin Multiplatform Gradle Plugin</PlatformLink>
- <PlatformLink to="/debug-symbols">Add readable stack traces to your errors</PlatformLink>
```
